cadc-storage-adapter-ad/src/intTest/java/org/opencadc/inventory/storage/ad/AdStorageAdapterGetTest.java

```diff
@@ -149,6 +168,30 @@ public void testGetIgnoresStorageBucket() {
             Assert.fail("Unexpected exception");
         }
     }
+
+    @Test
+    public void testGetVOSpac() throws Exception {
+        final AdStorageAdapter testAdapter = new AdStorageAdapter();
+
+        final URI expectedVOSpacChecksum = URI.create("md5:b05403212c66bdc8ccc597fedf6cd5fe");
+        Subject.doAs(testSubject, (PrivilegedExceptionAction<Object>) () -> {
+                final OutputStream outputStream = new ByteArrayOutputStream();
+                final DigestOutputStream digestOutputStream = new DigestOutputStream(outputStream, MessageDigest
+                        .getInstance(AdStorageAdapterGetTest.DIGEST_ALGORITHM));
+                final ByteCountOutputStream byteCountOutputStream = new ByteCountOutputStream(digestOutputStream);
+                final MessageDigest messageDigest = digestOutputStream.getMessageDigest();
+
+                final StorageLocation storageLocation = new StorageLocation(TEST_VOSPAC_URI);
+                storageLocation.storageBucket = AdStorageQuery.DISAMBIGUATE_PREFIX + "VOSpac:b054";
```

Comment:
I don't get why the test StorageLocation includes the DISAMBIGUATE. That `x-` is used in cases where one archive name is a prefix for another (eg GEM and GEMINI). It harmless since it gets removed by `bucket2archive` but it seems misleading here... makes one think it is necessary when it isn't.